operators/src/pro/cache/tile_cache.rs

```diff
@@ -612,14 +612,21 @@ mod tests {
 
     #[tokio::test]
     async fn it_evicts_lru() {
+        let cache_entry = CacheEntry {
+            query: query_rect(),
+            tiles: CachedTiles::U8(vec![create_tile()].into()),
+        };
+        let size_of_cache_entry = cache_entry.byte_size();
+        let total_cache_size = size_of_cache_entry * 4;
```

Comment:
why are you multiplying by for if the next line says "set limit s.t. three tiles fit"? :thinking:

Reply:
[add4280](https://github.com/geo-engine/geoengine/pull/800/commits/add42802cbaa765a87cbd6be32b7ba56a2d0481f)

I found the bug. 

I think the calculation of the size still needs some more work.